README.md

```diff
@@ -24,9 +24,10 @@ To Do:
 The configuration is done via environment variables stored in the `config.txt` file.
 
 For local development, copy the `config.txt.sample` file to `config.txt` to have a
-starting point. Then set the `OPENAI_API_KEY` variable to a valid OpenAI API key to
+starting point. Set the `OPENAI_API_KEY` variable to a valid OpenAI API key to
 enable that service. Or, otherwise set that variable the appropriate way when
-deploying the service.
+deploying the service. Also, retrieve `aiproxy/aiproxy/aiproxy_api_key` from the AWS Secrets Manager
```

Comment:
I think this would be a good place to mention that once you do this, you'll also need to set the aiproxy_api_key to the same value in locals.yml in the code-dot-org repo.